nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/tooltips/nifi-tooltip.directive.ts

```diff
@@ -58,11 +59,29 @@ export class NifiTooltipDirective<T> implements OnDestroy {
         }
     }
 
+    @HostListener('mousemove')
+    mouseMove() {
+        if (this.overlayRef?.hasAttached() && this.tooltipDisabled) {
+            this.overlayRef?.detach();
+        } else {
+            this.attach();
+        }
+    }
+
+    @HostListener('mouseup')
+    mouseup() {
+        this.attach();
```

Comment:
Nice touch but i think this should also be guarded. It currently can cause an error (similar to the above) if you mousedown outside of the target object and then mouseup over it. The tooltip will render based on the mouseenter and then try to attach another on the mouseup.
```suggestion
        if (!this.overlayRef?.hasAttached()) {
            this.attach();
        }
```

Reply:
Added guard